aspnetcore/fundamentals/static-files/samples/9.x/StaticFilesSample/Program.cs

```diff
@@ -44,7 +44,7 @@
 
 app.UseHttpsRedirection();
 
-app.UseStaticFiles(new StaticFileOptions
+app.UseStaticFiles(new StaticFileOptions  //
```

Comment:
Can't use `MapStaticAssets` here.

Reply:
They are compatible, but depending on how `app.UseRouting` is used, one or the other will take precedence. For a simple sample like this, I would avoid updating it to use `app.MapStaticAssets` and probably create a separate section for it.